cmd/registration-tracker/main.go

```diff
@@ -0,0 +1,377 @@
+package main
+
+/*
+	TODOs:
+		- Automate prefix discovery
+		- Restructure regtracker code base
+*/
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+	golog "log"
+	"net"
+	"os"
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"github.com/refraction-networking/conjure/pkg/phantoms"
+	"github.com/refraction-networking/conjure/pkg/station/geoip"
+	cj "github.com/refraction-networking/conjure/pkg/station/lib"
+	"github.com/refraction-networking/conjure/pkg/station/log"
+	"github.com/refraction-networking/conjure/pkg/transports"
+	"github.com/refraction-networking/conjure/pkg/transports/connecting/dtls"
+	"github.com/refraction-networking/conjure/pkg/transports/wrapping/min"
+	"github.com/refraction-networking/conjure/pkg/transports/wrapping/obfs4"
+	"github.com/refraction-networking/conjure/pkg/transports/wrapping/prefix"
+	pb "github.com/refraction-networking/conjure/proto"
+
+	"google.golang.org/protobuf/proto"
+)
+
+var (
+	ErrInvalidCC = errors.New("Invalid CC")
+	ErrIPtoCC = errors.New("IP to CC failed")
+	ErrIPtoASN = errors.New("IP to ASN failed")
+	ErrDupReg = errors.New("duplicate registration")
+)
+
+
+type transportStats struct {
```

Comment:
This is fine, but it may make more sense to have this just be a map of int64s, keyed on the transport.Name(), since that's what ends up happening in IncrementCount(). It would make it easier to add new transports (the name just has to be unique), and then printing it out is simply selecting what set of names/keys you want to output.